server/services/overviewPageService.test.ts

```diff
@@ -130,20 +178,47 @@ describe('OverviewPageService', () => {
       const prisonerNumber = '123123'
       const bookingId = 567567
 
-      const overviewPageService = new OverviewPageService(prisonApiClient)
+      const overviewPageService = new OverviewPageService(
+        prisonApiClient,
+        allocationManagerApiClient,
+        keyWorkerApiClient,
+      )
       const res = await overviewPageService.get({ ...PrisonerMockDataB, prisonerNumber, bookingId } as Prisoner)
       expect(res.personalDetails).toEqual(overviewPageService.getPersonalDetails(PrisonerMockDataB))
     })
   })
+
+  describe('getStaffContactDetails', () => {
+    it('should get the staff contact details for a prisoner', async () => {
+      const prisonerNumber = '123123'
+      const bookingId = 567567
+
+      const overviewPageService = new OverviewPageService(
+        prisonApiClient,
+        allocationManagerApiClient,
+        keyWorkerApiClient,
+      )
+      const res = await overviewPageService.get({ ...PrisonerMockDataB, prisonerNumber, bookingId } as Prisoner)
+      expect(res.staffContacts).toEqual(StaffContactsMock)
+    })
+  })
```

Comment:
This test is good for the "everything is Ok" route, but I think there's some logic within the service which handles a lot of edge cases/missing data cases that'd be good to make sure it works so that we don't have to manually check it every time (e.g. Logic around no case notes)

Reply:
Thanks for taking a look @danielburnley. I'll write some more tests. The utils are from the digital prison services repo, it would be good to wrap them in tests.